js/main.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+function start(canvas) {
+  const game = new Game(canvas.getGridDimensions().width, canvas.getGridDimensions().height);
+
+  game.init();
+  game.draw(canvas);
+
+  function frame() {
+    canvas.clear();
+    game.update();
+    game.draw(canvas);
+
+    if (game.gameOver) {
+      clearTimeout(setIntervalId);
+    }
+  }
+
+  const setIntervalId = setInterval(frame, 100);
+}
+
+const main = () => {
+  const c = document.getElementById('canvas');
+  const canvas = new Canvas(c);
+
+  const handler = (canvas) => {
+    document.removeEventListener('keypress', handler);
```

Comment:
The second argument of removeEventListener should be a "real" `function`. An arrow function is not considered exactly as a function (it has no context attached to it).
See https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener and https://developer.mozilla.org/en-US/docs/Web/API/EventListener

Reply:
I don't know why I said that, if you don't need `this`, you can use an arrow function. Apologies :pray: